hazelcast/src/test/java/com/hazelcast/internal/util/StringUtilTest.java

```diff
@@ -248,6 +248,12 @@ public void when_removingNotExistingCharactersFromString_then_sameInstanceIsRetu
         assertSame("-------", StringUtil.removeCharacter("-------", '0'));
     }
 
+    @Test
+    public void when_uppercaseWithTurkishLocale_then_RootLocaleIsUsed(){
```

Comment:
super nit: Root should start with lower case letter

Also, shouldn't we re-set the old value of Locale.setDefault after the test?

Reply:
changed method name